scripts/workers/worker_util.py

```diff
@@ -0,0 +1,104 @@
+import logging
```

Comment:
Deleted this file as it was part of other PR

Reply:
Perfect!